src/Test/DoctrineMongoDbOdmTestCase.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Test;
+
+use Doctrine\ODM\MongoDB\Configuration;
+use Doctrine\ODM\MongoDB\DocumentManager;
+use Doctrine\ODM\MongoDB\Mapping\Driver\AnnotationDriver;
+use Doctrine\ODM\MongoDB\UnitOfWork;
+use MongoDB\Client;
+use MongoDB\Model\DatabaseInfo;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @see https://github.com/doctrine/mongodb-odm/blob/2.0.0-RC1/tests/Doctrine/ODM/MongoDB/Tests/BaseTest.php
+ */
+abstract class DoctrineMongoDbOdmTestCase extends TestCase
```

Comment:
Is there a difference with https://github.com/doctrine/mongodb-odm/blob/2.0.0-RC1/tests/Doctrine/ODM/MongoDB/Tests/BaseTest.php?
If not why not using it directly? If there is, shouldn't we document which one?

Reply:
We cannot use it directly, as it's `export-ignore`: https://github.com/doctrine/mongodb-odm/blob/a50624c38534a3f238353c83c12fb268eee61281/.gitattributes#L7

Reply:
It's exactly the same, except for the two mock filters which I've removed, as we don't need those and they introduce dependencies: https://github.com/doctrine/mongodb-odm/blob/2.0.0-RC1/tests/Doctrine/ODM/MongoDB/Tests/BaseTest.php#L80-L81

Reply:
Maybe we only need this: https://github.com/doctrine/DoctrineMongoDBBundle/blob/4.0.0-RC1/Tests/TestCase.php

I'll try.

Reply:
There's more or less already something like this in API Platform: https://github.com/api-platform/core/blob/master/src/Test/DoctrineMongoDbOdmSetup.php

Reply:
Okay... :laughing: Let's try to use it then.